R/acf_dm.R

```diff
@@ -85,14 +86,18 @@ assign_acf_class <- function(pth,
              sheet = list(
                "CCSettings" = list(operation_number = "ProviderStateID",
                                    family_id = "ParentsID",
-                                   child_id = "ChildrenID"),
+                                   child_id = "ChildrenID",
+                                   date = "ReportingDate"),
                "Parents"  = list(family_id = "ParentsID",
                                  family_zip = "FamilyZip",
-                                 family_fips_code = "FIPS"), 
+                                 family_fips_code = "FIPS",
+                                 date = "ReportingDate"), 
                "Children"  = list(child_id = "ChildrenID",
-                                  child_age = "Age"), 
+                                  child_age = "Age",
+                                  date = "ReportingDate"), 
                "Providers"  = list(operation_number = "Data.StateID",
-                                   provider_zip = "Data.ZipCode"))), 
+                                   provider_zip = "Data.ZipCode",
+                                   date = "Data.ReportingDate"))), 
```

Comment:
@sadierichardson why do we need to add in the Data.ReportingDate?

Reply:
@holmesjoli There are three different reporting dates in each quarter, so I think we need to match family to provider using reporting date as well. I also am unsure if the child ids are unique across or within reporting dates so I added it in to investigate. But we're not using the ACS data for much anymore so I don't think it matters.

Reply:
Okay! Just wanted to double-check!